server/start.go

```diff
@@ -249,7 +253,6 @@ func startStandAlone(ctx *Context, appCreator types.AppCreator) error {
 func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.AppCreator) error {
 	cfg := ctx.Config
 	home := cfg.RootDir
-	var cpuProfileCleanup func()
 
 	if cpuProfile := ctx.Viper.GetString(flagCPUProfile); cpuProfile != "" {
```

Comment:
For the in-process case, which is the 99% case, you have this logic duplicated. It should be removed entirely from `startInProcess`.

Reply:
nice catch, fixed.